### PR TITLE
Instructions to back-merge release branches into master

### PR DESCRIPTION
**Pull request by eparis**
This allows us to continue on branches like v0.14.1, v0.14.2, and tag on
that branch.  But it will merge those tags into master so git describe
picks up the changes.

### CODE REVIEW

**Comment by eparis** [comment]
@zmerlynn @lavalamp 

This discusses how to make sure master picks up tags from the release branch, so versioning picks up info from the release branch....

File: build/mark-new-version.sh [hunk 1]
```diff
@@ -129,4 +129,17 @@ if [[ "${VERSION_PATCH}" == "0" ]]; then
...
   echo "- Submit branch: ${current_branch} as a PR to ${release_branch}"
   echo "- Get someone to review and merge that PR"
+  echo "- Update so you see that merge in origin"
+  echo "   git remote update"
+  echo "- Create and check out a new branch based on master"
+  echo "   git checkout -b merge-${release_branch}-to-master origin/master"
+  echo "- Merge the ${release_branch} into your merge-${release_branch}-to-master branch"
+  echo "   git merge ${release_branch}"
+  echo "   - It's possible you have merge conflicts."
+  echo "     - You will you will almost always want to take what is in HEAD"
```
Comment (lavalamp) [inline build/mark-new-version.sh]: s/you will//

File: build/mark-new-version.sh [hunk 2]
```diff
@@ -129,4 +129,17 @@ if [[ "${VERSION_PATCH}" == "0" ]]; then
...
+  echo "   - If you are not SURE how to solve these correctly, ask for help."
+  echo "   - It is possible to break other people's work if you didn't understand why"
+  echo "     the conflict happened and the correct way to solve it."
+  echo "- Submit merge-${release_branch}-to-master as a PR to master"
```
Comment (lavalamp) [inline build/mark-new-version.sh]: s/Submit/Send/ might be a little clearer?

**Comment by lavalamp** [comment]
Should perhaps mention in that wall of text the reason for those steps.

**Comment by eparis** [comment]
@lavalamp Hopefully fixed.  PTAL

**Comment by lavalamp** [comment]
LGTM

**Pull request merged**